http/headers/access-control-expose-headers.json

```diff
@@ -47,6 +47,54 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "wildcard": {
+          "__compat": {
+            "description": "Wildcard (<code>*</code>)",
+            "support": {
+              "chrome": {
+                "version_added": "63"
```

Comment:
It looks like this one didn't make it to Chrome until 65

https://storage.googleapis.com/chromium-find-releases-static/59a.html#59a2bc8acbe0926b70b5f926ae6e2383fc6dd532

I suspect some of the other browsers here need updating as a result

Reply:
Woah, great catch! Astonishing how often the info on the Chrome bug tracker is just wrong.

Reply:
Yeah, it is funny how inconsistent it is. However, I wasn't clear here: only expose headers was in 65; the others did ship in 63.

Reply:
Thanks, should have read the change set more carefully. I've set those other two back to 63.